SB2018/2018-03-09.md

```diff
@@ -0,0 +1,20 @@
+---
+to: All Teams
+subject: New SourceBots Kit Update
+---
+
+Hi!
+
+We're happy to announce a new update to our kits!
+
+This release contains many improvements to vision:
+- Completely re-do vision to greatly increase its accuracy (now all Cartesian values have an error within 15% of their total distance)
+- Greatly improve marker detection, it is now much less dependent on lighting conditions.
+
+A more detailed changelog for this release is available from [the docs](dl-link).
+
+To update your kit, you will need to flash your Pi's SD card. Instructions on [how to flash an SD card](sd-card-flashing), and the [download link](dl-link), are available from [the docs](updating-pi). If you have any issues updating, be sure to consult the documentation, or contact us by emailing techsupport@sourcebots.co.uk
```

Comment:
Explanation: We have to explicitly show this email without a link because my markdown processor (and the one @RealOrangeOne uses) doesn't handle if the email is a link.